src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Bundle\FrameworkBundle\Command\TranslationUpdateCommand;
+use Symfony\Component\Filesystem\Filesystem;
+
+class TranslationUpdateCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
@stof So just to clarify the workflow for adding a new test script:
* Find the oldest supported branch that contains the file to be tested.
    In this case, the branch is ``2.3``, but in the future if ``2.3`` is not supported any more, 
    the branch might be ``2.7`` for example.

*  Add the test script in that branch

Is that correct?